.env

```diff
@@ -0,0 +1 @@
+version=54e59e0adaa690fcd5ae9808a70d6e0bce2fe165
```

Comment:
notate in the README that the developer must create a .env file, unless this is auto generated for you somehow.

typically a .env's are ignored from source control, but if it's just taking a version (release version?) then that's okay here.